docs/tables/aws_s3tables_table.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Steampipe Table: aws_s3tables_table - Query AWS S3 Tables Tables using SQL"
+description: "Allows users to query AWS S3 Tables Tables, providing information about the configuration, settings, and properties of your S3 tables."
+folder: "S3Tables"
+---
+
+# aws_s3tables_table
```

Comment:
Please follow the format mentioned in https://github.com/turbot/steampipe-plugin-aws/blob/main/docs/tables/aws_amplify_app.md#table-aws_amplify_app---query-aws-amplify-apps-using-sql